### DISCUSSION TITLE
Step Parameter - Where to find??

### DISCUSSION THREAD

**Discussion by Pgmiranda1** [supermerill/SuperSlicer#1901] [Community contribution]
Hi.
Where do I find Step Parameter?
I try to do Retraction calibration and i found this:
![image](https://user-images.githubusercontent.com/94395539/142056622-73a0740f-ea8e-4c9a-828f-3dd2032da81a.png)

**Comment by supermerill** [comment]
![image](https://user-images.githubusercontent.com/6536403/142057481-beca8acb-5a27-4da6-ac64-6b045b2976ad.png)

**Reply by Pgmiranda1** [reply]
Thanks for your help.
(Sorry for my bad English).
Congratulations on the amazing work
